gp2gp-translator/src/main/java/uk/nhs/adaptors/pss/translator/amqp/PssQueueListener.java

```diff
@@ -0,0 +1,29 @@
+package uk.nhs.adaptors.pss.translator.amqp;
+
+import javax.jms.Message;
+import javax.jms.TextMessage;
+
+import org.springframework.jms.annotation.JmsListener;
+import org.springframework.stereotype.Component;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import lombok.AllArgsConstructor;
+import lombok.SneakyThrows;
+import uk.nhs.adaptors.pss.translator.amqp.task.TaskDefinition;
+
+@Component
+@AllArgsConstructor
```

Comment:
Can we replace AllArgs with @RequiredArgsConstructor(onConstructor = @__(@Autowired)) ?